src/locale/locales/de/messages.po

```diff
@@ -4505,77 +4319,77 @@ msgstr "Du hast keine Listen."
 
 #: src/view/screens/ModerationBlockedAccounts.tsx:132
 msgid "You have not blocked any accounts yet. To block an account, go to their profile and selected \"Block account\" from the menu on their account."
-msgstr ""
+msgstr "Sie haben noch keine Konten blockiert. Um ein Konto zu blockieren, gehen Sie zu seinem Profil und wählen Sie \"Konto blockieren\" aus dem Menü auf seinem Konto."
 
 #: src/view/screens/AppPasswords.tsx:87
 msgid "You have not created any app passwords yet. You can create one by pressing the button below."
 msgstr "Du hast noch keine App-Passwörter erstellt. Du kannst eines erstellen, indem du auf die Schaltfläche unten klickst."
 
 #: src/view/screens/ModerationMutedAccounts.tsx:131
 msgid "You have not muted any accounts yet. To mute an account, go to their profile and selected \"Mute account\" from the menu on their account."
-msgstr ""
+msgstr "Sie haben noch keine Konten stummgeschaltet. Um ein Konto stumm zu schalten, gehen Sie zu dessen Profil und wählen Sie \"Konto stummschalten\" aus dem Menü auf dessen Konto."
 
 #: src/view/com/modals/ContentFilteringSettings.tsx:175
 msgid "You must be 18 or older to enable adult content."
-msgstr ""
+msgstr "Sie müssen 18 Jahre oder älter sein, um Inhalte für Erwachsene zu aktivieren."
 
 #: src/screens/Onboarding/StepModeration/AdultContentEnabledPref.tsx:103
 msgid "You must be 18 years or older to enable adult content"
-msgstr ""
+msgstr "Sie müssen 18 Jahre oder älter sein, um Inhalte für Erwachsene zu aktivieren."
 
 #: src/view/com/util/forms/PostDropdownBtn.tsx:98
 msgid "You will no longer receive notifications for this thread"
-msgstr ""
+msgstr "Du wirst keine Mitteilungen mehr für diesen Thread erhalten"
 
 #: src/view/com/util/forms/PostDropdownBtn.tsx:101
 msgid "You will now receive notifications for this thread"
-msgstr ""
+msgstr "Sie erhalten nun Mitteilungen für dieses Thread"
```

Comment:
Informal:

```suggestion
msgstr "Du erhälst nun Mitteilungen für dieses Thread"
```

`Thread` can be kept, I guess. That word is common nowadays.

Reply:
Yeah the previous translators seemed to think that `Thread` was OK not to translate.